src/design-system/tokens.scss

```diff
@@ -32,16 +32,14 @@
   --x-size-min-width-grid-item: var(--x-size-base-20);
   --x-size-gap-grid: var(--x-size-base-07) var(--x-size-base-05);
   --x-size-padding-grid: var(--x-size-base-06) 0;
+  .x-partial-result {
```

Comment:
this is not right, but if i change it without affecting just the partial-result, other grid will be affected.

Reply:
Should the partials grid follow the same directives as the other grids (talking about the design here)?

Reply:
yeah, that would be nice. Maybe we can talk with cris about it